ssl/t1_enc.c

```diff
@@ -175,6 +175,18 @@ int tls_provider_set_tls_params(SSL *s, EVP_CIPHER_CTX *ctx,
     return 1;
 }
 
+
+static int EVP_CIPHER_iv_length_within_key_block(const EVP_CIPHER *c)
```

Comment:
Don't prefix this function with `EVP_CIPHER_`. We typically use this capitalisation to indicate that it is an external function. Since this is a tls specific calculation, I'd prefer something like `tls_iv_length_within_key_block`...or perhaps more simply `tls_fixed_iv_length`